services/director/src/simcore_service_director/client_session.py

```diff
@@ -1,41 +1,22 @@
-from aiohttp import ClientSession, ClientTimeout
-from common_library.json_serialization import json_dumps
+import httpx
 from fastapi import FastAPI
-from servicelib.utils import (
-    get_http_client_request_aiohttp_connect_timeout,
-    get_http_client_request_aiohttp_sock_connect_timeout,
-    get_http_client_request_total_timeout,
-)
 
 
 def setup_client_session(app: FastAPI) -> None:
     async def on_startup() -> None:
-        # SEE https://github.com/ITISFoundation/osparc-simcore/issues/4628
-
-        # ANE: it is important to have fast connection handshakes
-        # also requests should be as fast as possible
-        # some services are not that fast to  reply
-        timeout_settings = ClientTimeout(
-            total=get_http_client_request_total_timeout(),
-            connect=get_http_client_request_aiohttp_connect_timeout(),
-            sock_connect=get_http_client_request_aiohttp_sock_connect_timeout(),
-        )
-        session = ClientSession(
-            timeout=timeout_settings,
-            json_serialize=json_dumps,
-        )
+        session = httpx.AsyncClient(transport=httpx.AsyncHTTPTransport(http2=True))
```

Comment:
registry accepts http/2?

Reply:
well it does not reject it so why not use it?